packages/contracts/contracts/voting/addresslist/Addresslist.sol

```diff
@@ -29,15 +33,21 @@ abstract contract Addresslist is TimeHelpers {
     /// @param members The array of member addresses to be removed.
     event AddressesRemoved(address[] members);
 
-    /// @notice Checks if an account is on the address list at given block number.
+    /// @notice Checks if an account is on the address list at a specific block number.
     /// @param _account The account address being checked.
     /// @param _blockNumber The block number.
-    function isListed(address _account, uint256 _blockNumber) public view returns (bool) {
-        if (_blockNumber == 0) _blockNumber = getBlockNumber64() - 1;
-
+    /// @return Whether the account is listed at the specified block number.
+    function isListedAtBlock(address _account, uint256 _blockNumber) public view returns (bool) {
```

Comment:
OZ also names their method explicitly as `getAtBlock()` and `latest()` to obtain the respective checkpoint.
In your solution, if we as developers forget to give the second `_blockNumber` argument and the code wrongly compiles, this can create huge problems which we might overlook.
I am very much for letting things fail as early as possible, at best during compile time.

Reply:
Resolved after call with @novaknole